tests/core/limits_test.py

```diff
@@ -11,46 +11,46 @@
 def test_find_limits():
     x = sc.arange('x', 11.0, unit='m')
     lims = find_limits(x)
-    assert sc.identical(lims[0], sc.scalar(0.0, unit='m'))
-    assert sc.identical(lims[1], sc.scalar(10.0, unit='m'))
+    assert sc.identical(lims[0], sc.scalar(-0.5, unit='m'))
```

Comment:
A little annoying that if we change the size of the padding, we have to change all the tests...